src/libs/Aggregator/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { createHash } from 'crypto';
```

Comment:
browser environment does not have this module.

Reply:
fixed